cyprecice/cyprecice.pyx

```diff
@@ -5,11 +5,8 @@
 The python module precice offers python language bindings to the C++ coupling library precice. Please refer to precice.org for further information.
 """
 
+cimport cyprecice
```

Comment:
Does this mean we have to rename the package from `pyprecice` to `cyprecice` or is this just some internal naming?

Reply:
This is just internal naming. See https://github.com/precice/python-bindings/blob/fix_pkg/precice/__init__.py. Here the python package `precice` imports everything from `cyprecice`.